.travis.yml

```
@@ -1,4 +1,8 @@
 language: go
 
 go:
- - 1.11.4
+ - 1.12
```

Let's try to test with both versions (not tested) 
```suggestion
 - 1.11.4 
 - 1.12
```